Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/invalid/SubmissionTestResultInvalidFragment.kt

```diff
@@ -0,0 +1,67 @@
+package de.rki.coronawarnapp.ui.submission.testresult.invalid
+
+import android.app.AlertDialog
+import android.os.Bundle
+import android.view.View
+import android.view.accessibility.AccessibilityEvent
+import androidx.activity.OnBackPressedCallback
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentSubmissionTestResultInvalidBinding
+import de.rki.coronawarnapp.util.DialogHelper
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class SubmissionTestResultInvalidFragment : Fragment(R.layout.fragment_submission_test_result_invalid), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    private val vm: SubmissionTestResultInvalidViewModel by cwaViewModels { viewModelFactory }
+
+    private val binding: FragmentSubmissionTestResultInvalidBinding by viewBindingLazy()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val backCallback: OnBackPressedCallback = object : OnBackPressedCallback(true) {
```

Comment:
We do not need to override the back navigation behavior unless we need to perform a specific task, like showing a dialog. The back stack should be cleared in the action leading to this fragment, eg. popping up to main fragment, so pressing back naturally lands on main fragment